server/src/integrations/message-queue/message-queue.constants.ts

```diff
@@ -0,0 +1,5 @@
+export const QUEUE_DRIVER = Symbol('QUEUE_DRIVER');
+
+export enum MESSAGE_QUEUES {
```

Comment:
@mahendraHegde Enum shouldn't be in capitalize and snake case, but in PascalCase even if it's an injection token !
Maybe we should do something like `@nestjs/bull` and create a custom decorator `@InjectQueue` that is checking if parameter is properly type and generating a better injection token based on the queue name.

Reply:
yes, i thought of custom decorator, it makes sense if we do modifications on the token(nest adds a prefix), but in our case, its better to use enum directly just to avoid any mistakes in types and avoid unnecessary logic to check type again.  

for the consumer class, for `@nest/bull` it makes sense because bullmq is designed to use separate classes for consumer and producer, but pg-boss uses single class for both, so i decided to keep the single class, consumer class also can inject the queue class and use the consumer method instead.

let me know what do you think.